content/docs/command-reference/repro.md

```diff
@@ -262,31 +263,42 @@ The answer to universe is 42
 - The Hitchhiker's Guide to the  Galaxy
 ```
 
-Now, using the `--downstream` option results in the following output:
+And update the `process.py` file to count the number of digits.
+
+```python
+import sys
+num_digits = 0
+with open(sys.argv[1], 'r') as f:
+    for number in f:
+        num_digits += len(number) - 1
+print("Number of digits:",end=" ")
+print(num_digits)
+```
+
+Now, using the `--downstream` option with `count` as a target stage, results in
+the following output:
 
 ```dvc
-$ dvc repro --downstream
-Data and pipelines are up to date.
+$ dvc repro --downstream count
+Running stage 'count' with command:
+        python3 process.py numbers.txt > count.txt
+Updating lock file 'dvc.lock'
 ```
 
-The reason being that the `text.txt` file is not a dependency in the last stage
-of the pipeline, used as the default target by `dvc repro`. `text.txt` is a
-dependency of the `filter` stage, which happens earlier (shown in the figure
-below), so it's skipped given the `--downstream` option.
+The reason being that the `text.txt` file is a dependency in the `filter` stage
+of the pipeline which happens before the `count` stage (shown in the following
+figure) and hence did not get updated.
```

Comment:
The reason for what? This explanation needs more detail.
"and hence did not get updated" what does this refer to? one of the stages? which one? the pipeline?
should be broken up into several sentences too.